views/albums.hbs

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    {{#each response.items}}
+    <div>
+            <a href="/albums/{{tracks}}">View Tracks </a>
```

Comment:
You're sending the user back to same page `albums` instead of sending to `tracks`:
```
 <a href="/tracks/{{id}}">View Tracks </a>
```